flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why restricting architectures?

Reply:
I don't think it works on other architectures, and I don't feel like testing for other architectures.

Reply:
surely this won't be tested with that reasoning.

Reply:
If I understand correctly, this flathub.json file will prevent flathub's build system from trying to build other arches? 

The idea was to prevent flathub from unnecessarily trying to build versions that will fail anyway, and as a way to show that I don't intend to support other arches in the immediate future. I might be wrong in my thinking. 

What are your suggestions?

Reply:
why don't give it a try? Without a build no one can test it. But unlike I suggested you deleted part of what's needed to build on aarch64.